tasks/Game of Clue - Terrorist Attack.md

```diff
@@ -0,0 +1,38 @@
+# Summary
+Participants played a 'who-done-it' game like Clue or Cluedo in wich the task involved piecing together clues, or facts to connect the dots. Specifically, participants were faced with four logically independent subproblems to solve: (a) who would carry out the attack (group involved), (b) what would be the target (e.g., an embassy or a church), (c) where the attack would take place (country), and (d) when the attack would take place (with four interdependent components—month, day, hour, and time of day (a.m. versus p.m.)). 
+
+# References
+Main Paper: https://pubsonline.informs.org/doi/full/10.1287/orsc.2015.0980#_i34
+
+Supplemental Materials: https://pubsonline.informs.org/doi/suppl/10.1287/orsc.2015.0980/suppl_file/orsc.2015.0980-sm-online_appendix_.pdf
+
+# Stimuli
+## The visual components
+See supplemetal materials PDF for images of computer interface 
+
+
+# Procedure
+## Steps
+1. The experiment was carried out in a laboratory setting, with each participant seated at a computer in a private carrel. Each experimental run lasted 25 minutes. Each experimental session began with an instructional video explaining the platform and the task in uniform fashion to every subject across all sessions. To control for individual aptitude, each person then took a pretest with the same format as the experimental task, with a similar but smaller-scale problem and without interaction with other participants.
+2. Subjects could take part in up to three runs of the experiment. Within any given run, all subjects had the same level of experience with the experiment. That is, if a subject were taking part in a second run, then all of the other subjects in that run would also be taking part for the second time. Holding this experience constant eliminated the possibility of spurious correlations between learning and network structure. In the multivariate analyses, we include a round variable that indicates the experience of the participants in number of runs through the experiment.
+> Before each run began, a network treatment was chosen at random and study subjects were randomly assigned to a position in that network that was uncorrelated to their physical location in the laboratory. Participants were assigned a pseudonym to further obscure their identities from each other; the pseudonyms were shuffled before each round.
+4. Participants were given two clues at the start of each round and were allowed to search for more clues once per minute. 
```

Comment:
They could share clues they had one at a time with one or more of their neighbors, and could add free-text annotations to these shared clues